ex04.ex

```diff
@@ -75,7 +83,26 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
+  def add_if_even(value, result) do
```

Comment:
# 6

I have two issues here. First, you're traversing the list twice. Second, there's a lot of duplication between add_if_even and add_if_odd. You could have parameterized a more general function